sale_product_multi_add/wizards/sale_import_products.py

```diff
@@ -45,6 +45,7 @@ def _get_line_values(self, sale, item):
             }
         )
         sale_line.product_id_change()
+        sale_line._onchange_discount()
```

Comment:
Can you please add a `FIXME` comment for using `onchange_helper` here to avoid missing running all needed onchanges?

Reply:
Thank you, @simahawk, for your review. I've just come across the onchange_helper module, but I'm unsure if I'm using it correctly. I would appreciate it if you could take a look and provide your feedback.

cc @TumbaoJu